packages/graphile-build-pg/src/utils.js

```diff
@@ -44,3 +44,24 @@ export const camelCase = formatInsideUnderscores(camelCaseAll);
 export const constantCase = formatInsideUnderscores(constantCaseAll);
 export const upperCamelCase = (str: string): string =>
   upperFirst(camelCase(str));
+
+export const parseTags = (str: string) => {
+  return str.split(`\n`).reduce((prev, curr) => {
+    const match = curr.match(/^@[a-z]+ ?/);
+    return match &&
+    prev.text === "" &&
+    curr.split(" ")[0] === match[0].split(" ")[0]
+      ? Object.assign({}, prev, {
+          tags: Object.assign({}, prev.tags, {
+            [match[0].substr(1).trim()]:
+              match[0] === curr ? true : curr.replace(match[0], ""),
```

Comment:
Let's turn this into an array if there's multiple matches. i.e.

```
@jsonField date timestamp
@jsonField name text
@jsonField episode enum ONE=1 TWO=2
```

Reply:
Should produce `{jsonField: ["date timestamp", "name text", "episode enum ONE=1 TWO=2"]}`

Reply:
(This is an enhancement to the previous spec in order to enable https://github.com/postgraphql/postgraphql/issues/642#issuecomment-348731162 )

Reply:
Makes sense.  Should we allow mixed case tag names like `jsonField`?  The original spec was `/@[a-z]+/`.

Reply:
Definitely; that was a bug in my spec, sorry! Probably underscores too. Maybe even numbers... enforce it starts with [a-zA-Z] though.